hack/setup-local.sh

```diff
@@ -1,8 +1,9 @@
 kind create cluster --name kyverno-cluster
-kind get kubeconfig --name kyverno-cluster > ./kube.config
-kubectl create --context kind-kyverno-cluster -f https://raw.githubusercontent.com/kyverno/kyverno/main/definitions/release/install.yaml
+kubectl wait nodes/kyverno-cluster-control-plane --for=condition=ready --timeout=5m > /dev/null
+kind get kubeconfig --name kyverno-cluster > $(pwd)/kube.config
 kubectl create --context kind-kyverno-cluster -f https://raw.githubusercontent.com/giantswarm/apiextensions/b9a6bed05850045530eaf8ba7fd01941de6c8525/helm/crds-common/templates/giantswarm.yaml
 kubectl create --context kind-kyverno-cluster -f https://raw.githubusercontent.com/kubernetes-sigs/cluster-api/release-0.3/config/crd/bases/cluster.x-k8s.io_clusters.yaml
 kubectl create --context kind-kyverno-cluster -f https://raw.githubusercontent.com/kubernetes-sigs/cluster-api/release-0.3/config/crd/bases/cluster.x-k8s.io_machinedeployments.yaml
 kubectl create --context kind-kyverno-cluster -f https://raw.githubusercontent.com/kubernetes-sigs/cluster-api-provider-aws/main/config/crd/bases/infrastructure.cluster.x-k8s.io_awsclusters.yaml
 kubectl create --context kind-kyverno-cluster -f https://raw.githubusercontent.com/kubernetes-sigs/cluster-api-provider-aws/main/config/crd/bases/infrastructure.cluster.x-k8s.io_awsclusterroleidentities.yaml
+kubectl create --context kind-kyverno-cluster -f https://raw.githubusercontent.com/kyverno/kyverno/main/definitions/release/install.yaml
```

Comment:
This needed to be reordered following upstream issue: https://github.com/kyverno/kyverno/issues/2094

Reply:
Does `abs` have logic to wait for kyverno to be up? If not, you could wait here. Something like https://github.com/giantswarm/coredns-warnlist-plugin/blob/f1d8ff108254e4f07968b718e933593de97ecd5d/.circleci/config.yml#L98

Reply:
app-build-suite (soon to be replaced with app-test-suite for testing) usually waits until the app cr shows status deployed

https://github.com/giantswarm/app-test-suite/blob/master/app_test_suite/steps/base_test_runner.py#L315

Reply:
I thought about waiting here but currently it will just make the test run longer - kyverno is up way before the tests start. 

The problem is that kyverno is not the app-cr and therefor we wait for the setup of the kyverno-policies.

All in all not a big deal currently.

Reply:
The problem with the bug I discovered was, that kyverno does currently not reload CRDs after it is started. This caused kyverno to fail if it was ready before all the CRDs were applied.